src/core/src/main/java/org/apache/jmeter/util/JsseSSLManager.java

```diff
@@ -213,14 +213,15 @@ private SSLContext createContext() throws GeneralSecurityException {
         JmeterKeyStore keys = this.getKeyStore();
         managerFactory.init(null, defaultpw == null ? new char[]{} : defaultpw.toCharArray());
         KeyManager[] managers = managerFactory.getKeyManagers();
-        KeyManager[] newManagers = new KeyManager[managers.length];
+        int managersCount = managers.length;
+        KeyManager[] newManagers = new KeyManager[managersCount];
 
         if (log.isDebugEnabled()) {
             log.debug("JmeterKeyStore type: {}", keys.getClass());
         }
 
         // Now wrap the default managers with our key manager
-        for (int i = 0; i < managers.length; i++) {
+        for (int i = 0; i < managersCount; i++) {
```

Comment:
Could it better be changed to for-each loop instead of factoring `managersCount` variable?

Reply:
Hi @vlsi agreed, but the new version will still use index since there is insertion on the array given the index. Are you ok with this change?

Example:
```
int indexManager = 0;
        for (KeyManager manager : managers) {
            if (manager instanceof X509KeyManager) {
                newManagers[indexManager] = new WrappedX509KeyManager((X509KeyManager) manager, keys);
            } else {
                newManagers[indexManager] = manager;
            }
            indexManager++;
        }
```

Reply:
Yeah, I missed the index is needed there.
However, `managers` variable does not mutate, so JVM should be able to inline `managers.length` here.

Frankly, I find `managers.length` much easier to read than the variant with an extra variable, and I bet extracting `managers.length` to a variable does not improve performance. So I would suggest keeping the old code as is.

Reply:
Old code was reverted @vlsi